src/components/molecules/BadgePreview/BadgePreview.tsx

```diff
@@ -30,6 +32,7 @@ const defaultValuesForBadgePreviewProps = {
   subline: '',
   description: '',
   animationOnHover: false,
+  animationEffects: ['wobble', 'grow'] as Effects[],
```

Comment:
This shouldn't be a list of animations on hover, because if you set more than 1 they don't work together. It is one or the other.
I think we can change that to:
```suggestion
  animationOnHover: BadgePreviewAnimation,
```
```
export enum BadgePreviewAnimation {
  NONE = 'NONE',
  GROW = 'GROW',
  GROW_WITH_TRANSITION = 'GROW_WITH_TRANSITION',
  WOBBLE = 'WOBBLE'
}
```

Reply:
I dont see the value of an enum, for me it is too verbose... But you are right that doesn't work together I was planning to push a workaround for it.

Reply:
Another reason in favor of enums is that you can refactor it very easily if needed. Handling strings makes you look for all literal matches in the project if you want to refactor or change that.